From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Mon, 11 Jul 2022 21:33:53 +0900
Subject: "py7zr" uses PyCryptodome

Forwarded: not-needed

This test checks the Cryptodome class was replaced old Crypto class.
py7zr requires Cryptodome's extended features like AES.
---
 src/calibre/test_build.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/calibre/test_build.py b/src/calibre/test_build.py
index f64ca06..26b32bd 100644
--- a/src/calibre/test_build.py
+++ b/src/calibre/test_build.py
@@ -47,7 +47,7 @@ class BuildTest(unittest.TestCase):
                     self.assertTrue(False, f'Failed to load DLL {x} with error: {err}')
 
     def test_pycryptodome(self):
-        from Crypto.Cipher import AES
+        from Cryptodome.Cipher import AES
         del AES
 
     @unittest.skipUnless(islinux, 'DBUS only used on linux')
